lib/takeaway.rb

```diff
@@ -0,0 +1,63 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'texter'
+
+class Takeaway
+  attr_reader :menu, :order
+
+  def initialize(order = [], menu = Menu.new, texter = Texter.new)
+    @order = order
+    @menu = menu
+    @texter = texter
+  end
+
+  def view_menu
+    @menu.view
+  end
+
+  def add(item, quantity = 1)
+    raise "Please enter a item on the menu" if check_item(item) == false
+
+    @order << { Item: item, Quantity: quantity, Price: @menu.falafel_menu.fetch(item) }
+  end
+
+  def check_item(item)
+    @menu.falafel_menu.key?(item)
+  end
```

Comment:
This could perhaps be a private method to avoid someone trying to call it where they shouldn't